MarkdownConverter/Spec/Span.cs

```diff
@@ -1,13 +1,15 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace MarkdownConverter.Spec
+﻿namespace MarkdownConverter.Spec
 {
     internal class Span
     {
-        public int start, length;
-        public Span(int start, int length) { this.start = start; this.length = length; }
-        public int end => start + length;
+        public int Start { get; }
+        public int Length { get; }
+        public int End => Start + Length;
+
+        public Span(int start, int length)
```

Comment:
Consider this form:

```csharp
public Span(int start, int length) => (Start, Length) = (start, length);

Reply:
Yes, I always have mixed feelings about that. I really like it, but I wonder whether it's sufficiently well-known at the moment. Happy to follow your suggestion though; will do in a follow-up PR (just for the sake of convenience for where I am right now...)